packages/client/src/monaco-language-client.ts

```diff
@@ -3,36 +3,25 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
 import {
-    BaseLanguageClient, MessageTransports, LanguageClientOptions,
-    StaticFeature, DynamicFeature
-} from "vscode-languageclient/lib/common/client";
-import { TypeDefinitionFeature } from "vscode-languageclient/lib/common/typeDefinition";
-import { ConfigurationFeature as PullConfigurationFeature } from "vscode-languageclient/lib/common/configuration";
-import { ImplementationFeature } from "vscode-languageclient/lib/common/implementation";
-import { ColorProviderFeature } from "vscode-languageclient/lib/common/colorProvider";
-import { WorkspaceFoldersFeature } from "vscode-languageclient/lib/common/workspaceFolders";
-import { FoldingRangeFeature } from "vscode-languageclient/lib/common/foldingRange";
-import { CallHierarchyFeature } from "vscode-languageclient/lib/common/callHierarchy";
-import { ProgressFeature } from "vscode-languageclient/lib/common/progress";
-import { SemanticTokensFeature } from "vscode-languageclient/lib/common/semanticTokens";
+    BaseLanguageClient, MessageTransports, LanguageClientOptions
+} from "vscode-languageclient";
 import * as p2c from 'vscode-languageclient/lib/common/protocolConverter';
 import * as c2p from 'vscode-languageclient/lib/common/codeConverter';
-import { IConnectionProvider, IConnection } from './connection';
-import { DeclarationFeature } from "vscode-languageclient/lib/common/declaration";
+import { IConnectionProvider } from './connection';
 import { CompletionParams, WillSaveTextDocumentParams } from './services'
 
 export * from 'vscode-languageclient/lib/common/client';
+import type * as vscode from 'vscode'
 
 export class MonacoLanguageClient extends BaseLanguageClient {
 
-    static bypassConversion = (result: any) => result || undefined;
+    static bypassConversion = (result: any, token?: vscode.CancellationToken) => token != null ? Promise.resolve(result || undefined) : (result || undefined);
```

Comment:
I'm not sure to understand what you mean, can you provide the code you want to be there?

Reply:
Shouldn't `token != null` be`token !== null`?

Reply:
Do you want me to change it?

Reply:
Ok, so it can only just be:
```js
token ? Promise.resolve(result || undefined) : (result || undefined);
```
because the Typescript says:
```js
token?: vscode.CancellationToken | undefined
```
So, it exists or doesn't.
But, of course with JavaScript someone can throw in `null`, so if it `null` or `undefined` should be treated the same, then just leave it code as it is and sorry for the confusion. If you think types are the law, then make it strict. :-)

Reply:
I'm used to strict eslint rules, so doing a if on something else than a boolean make me crazy now :scream:

Reply:
This is a boolean expression, if is defined, then it is true: `token ? Promise.resolve(result || undefined) : (result || undefined);`

Reply:
> Can we release this one or do you want to wait for the next version?

I am just now working on bringing the webpack config for the web client example back. If both are in, then we can release a new version. Should we consider investigating the new issue that came up before. WDYT?

Reply:
I mean a real new version `0.19.0` or shall we call it `1.0.0`?

Reply:
> This is a boolean expression, if token is defined, then it is true: token ? Promise.resolve(result || undefined) : (result || undefined);

I'm not sure to understand what you mean, `token` is a boolean?

> Should we consider investigating the new issue that came up before. WDYT

Are you talking about https://github.com/TypeFox/monaco-languageclient/issues/351? I don't think it's a new issue, but I'm currently investigating. There is probably no reason for it to block the next release tho

> I mean a real new version 0.19.0 or shall we call it 1.0.0?

I never understand why so many libraries are using a version number < 1 even after years (even monaco!).
But it's not a big deal, as you wish!

Reply:
Let's aim for 1.0.0. webpack example is almost working again. `monaco` workers are not working properly. Will fix this today. Release today or tomorrow.

Reply:
> You can rewrite this as such (token is undefined => false or defined => true)
That's what I meant when talking about `if on anything else than boolean`, relying on the trythiness of an object and the falsiness of undefined seems wrong and dangerous, no big deal tho since the whole project is done this way

Reply:
Please press the merge button if you like. 👍